test/config/reaper/bases/reaper.yaml

```diff
@@ -3,7 +3,7 @@ kind: Reaper
 metadata:
   name: reaper-test
 spec:
-  image: docker.io/thelastpickle/cassandra-reaper:2.2.5
+  image: docker.io/adejanovski/cassandra-reaper:413af30
```

Comment:
Should probably be in k8ssandra repo.

Reply:
Because of Reaper and Medusa being used outside of k8ssandra I believe the decision was made to keep them
in the TLP org. On that basis I don't see a need to publish the image under k8ssandra.

Reply:
But that's not in the TLP org either.

Reply:
I didn't mean to commit the PR like this, as said in my comment :) 

> We'll have to wait until a new Docker tag is released for Reaper before merging the PR, but it would be good to get an approval ASAP so that we can get this into 1.3.

I have mixed feelings about merging the PR on a tag that's not an actual release, but we'll probably have to wait a little bit more until we get a new release of Reaper. I may just create a tag with a name that's easy enough to spot in the tlp org and update this PR so it can be merged.

Reply:
I've moved to a tag on the tlp repo now. I'll try to remember to upgrade this when Reaper 2.3.0 gets released.

Reply:
So are we good to put this into 1.3.0 before Reaper 2.3.0 is out?

I just talked to Erik earlier and we're looking at a few more days for the incremental repair work, so if we were waiting for that before releasing Reaper we may want to just go ahead and put out a patch.

Reply:
yes, we can go ahead and merge it as is. I'd like us to get the incremental repair stuff in before releasing 2.3.0.

Reply:
Assuming Reaper 2.3.0 will be released before k8ssandra 1.3.0 then we can open a follow up PR to bump the default version. I am in favor of merging and updating in k8ssandra to give the changes some soak time before releasing 1.3.0.

Reply:
The trick here is that I don't think we're going to get reaper-2.3.0 out before k8ssandra-1.3.0.  So if we want to include this in k8ssandra-1.3.0 we'll need to use some patch version of reaper that we could get out sooner.

@adejanovski what do you think about releasing a v2.2.6 of Reaper that would pickup your work in https://github.com/thelastpickle/cassandra-reaper/pull/1096 and use that?

We could hopefully then get reaper-2.3.0 out post k8ssandra-1.3.0 and use that in k8ssandra-1.4.0 to pickup the incremental repair support?

Reply:
> @adejanovski what do you think about releasing a v2.2.6 of Reaper that would pickup your work in thelastpickle/cassandra-reaper#1096 and use that?

It's doable but slightly breaks the contract that patch releases are for fixes only.

Since we have no solid ETA for incremental repair improvements, I guess we could release 2.3.0 right now and delay other planned features to 2.4.0.
wdyt @jdonenine ?

Reply:
That works for me @adejanovski 

I'd vote we go ahead and release 2.3.0 and we'll come back with 2.4.0 for incremental repair when it's ready.

Reply:
For the record, so you don't think I've gone crazy, I was thinking a patch release because what we're generally trying to fix is a bug:  https://github.com/k8ssandra/reaper-operator/issues/56

But you're totally right that what we added to reaper itself, to address the bug over here in K8ssandra, isn't a bug fix.  So I think you've got the right suggestion with bump to 2.4.0.